package.json

```
@@ -51,6 +51,7 @@
     "debug": "^4.1.1",
     "p-map": "^4.0.0",
     "packageurl-js": "^1.0.0",
+    "shescape": "^1.6.0",
```

Pinned version please.
```suggestion
    "shescape": "1.6.1",
```